### ISSUE TITLE
MAP is too low.

### ISSUE THREAD

**Issue by zhangxiaohuixuhao** [eriklindernoren/PyTorch-YOLOv3#634]
Hi, I'm using well this repo in COCO dataset and my custom dataset. 

While doing my project with this repo, I had a two question about this.

1. When using the COCO dataset for training, in the first eight epochs, the MAP of each test is small. The MAP is about 0.023, is this normal?

2. When I use custom dataset for training, the following error occurred: ---------mAP not measured (no detections found by model). And I have changed the label to 'label_idx x_center y_center width height'.

Thank you for reading my issue.

**Comment by Flova** [comment]
Hey,

regarding your first point, this is a known issue as described in #513.

The second point is expected behavior because the model doesn't make useful predictions in the first few epochs and the evaluation gets skipped if no detections are available.

I hope this helps. Feel free to reopen for further questions.
